confidential-ledger-demo/server/confidentialledger.py

```diff
@@ -1,224 +1,90 @@
-# Copyright (c) 2021 Microsoft.
-# Copyright (c) 2021 Avanade Inc.
+# Import the Azure authentication library
 
+from azure.identity import DefaultAzureCredential
 
-from enum import Enum
+# import the data plane sdk
 
-from typing import Dict, NamedTuple, Iterable
-import argparse
+from azure.confidentialledger import ConfidentialLedgerClient
+from azure.confidentialledger.identity_service import (
+    ConfidentialLedgerIdentityServiceClient,
+)
 
-from dotenv import load_dotenv
-import requests
 import os
```

Comment:
We should reorder the imports - have you run pylint?

Reply:
will run and re-factor